includes/elements/container.php

```diff
@@ -174,6 +178,38 @@ private function render_video_background() {
 		</div><?php
 	}
 
+	/**
+	 * Render the Container's shape divider.
+	 * TODO: Copied from `section.php`.
+	 *
+	 * Used to generate the shape dividers HTML.
+	 *
+	 * @param string $side - Shape divider side, used to set the shape key.
+	 *
+	 * @return void
+	 */
+	private function render_shape_divider( $side ) {
```

Comment:
@StyleShit, suggestion: it can have common base.

Reply:
@matipojo 
Do we want to combine Container & Section shape divider controls using a Trait or something?

Reply:
No, we don't want to depend on each other